CdiBeanCaseTaskResolutionTest: Cause: Cannot resolve identifier 'caseVariableBean'

**Describe the bug**
The test `CdiBeanCaseTaskResolutionTest` fails on the Tomcat integration tests.

```
19-Mar-2025 03:35:41.851 INFO [Thread-4] org.operaton.commons.logging.BaseLogger.logInfo ENGINE-14020 JobExecutor[org.operaton.bpm.engine.impl.jobexecutor.RuntimeContainerJobExecutor] stopped job acquisition
19-Mar-2025 03:35:42.018 SEVERE [http-nio-48080-exec-5] org.operaton.commons.logging.BaseLogger.logError ENGINE-16004 Exception while closing command context: Unknown property used in expression: ${caseVariableBean.getNewValue(caseExecution)}. Cause: Cannot resolve identifier 'caseVariableBean'
	org.operaton.bpm.engine.ProcessEngineException: Unknown property used in expression: ${caseVariableBean.getNewValue(caseExecution)}. Cause: Cannot resolve identifier 'caseVariableBean'
		at org.operaton.bpm.engine.impl.el.JuelExpression.getValue(JuelExpression.java:65)
		at org.operaton.bpm.engine.impl.el.JuelExpression.getValue(JuelExpression.java:52)
		at org.operaton.bpm.engine.impl.el.ElValueProvider.getValue(ElValueProvider.java:41)
		at org.operaton.bpm.engine.impl.core.model.CallableElementParameter.getSource(CallableElementParameter.java:46)
		at org.operaton.bpm.engine.impl.core.model.CallableElementParameter.applyTo(CallableElementParameter.java:60)
		at org.operaton.bpm.engine.impl.core.model.CallableElement.getVariables(CallableElement.java:128)
		at org.operaton.bpm.engine.impl.core.model.CallableElement.getOutputVariables(CallableElement.java:115)
		at org.operaton.bpm.engine.impl.cmmn.behavior.ProcessOrCaseTaskActivityBehavior.getOutputVariables(ProcessOrCaseTaskActivityBehavior.java:63)
		at org.operaton.bpm.engine.impl.cmmn.behavior.ProcessOrCaseTaskActivityBehavior.transferVariables(ProcessOrCaseTaskActivityBehavior.java:45)

Caused by: jakarta.el.PropertyNotFoundException: Cannot resolve identifier 'caseVariableBean'
		at org.operaton.bpm.impl.juel.AstIdentifier.eval(AstIdentifier.java:88)
```

The test is disabled for now.

**To Reproduce**

```
./mvnw -Pdistro,distro-webjar,h2-in-memory,tomcat,distro-tomcat clean install -DskipTests

./mvnw -Pengine-integration,tomcat,h2 verify -Dsurefire.includes="**/CdiBeanCaseTaskResolutionTest.java"
```

**Expected behavior**
Test is working.

**Additional context**
Bug introduced by [#10dacdeaf89caf155908846a900ba6d7a16bc039](https://github.com/operaton/operaton/commit/10dacdeaf89caf155908846a900ba6d7a16bc039)

--- a/qa/integration-tests-engine/src/test/java/org/operaton/bpm/integrationtest/functional/cdi/CdiBeanCaseTaskResolutionTest.java
+++ b/qa/integration-tests-engine/src/test/java/org/operaton/bpm/integrationtest/functional/cdi/CdiBeanCaseTaskResolutionTest.java
@@ -40,7 +40,6 @@ import static org.assertj.core.api.Assertions.assertThat;
  *
  */
 @ExtendWith(ArquillianExtension.class)
-@Disabled("https://github.com/operaton/operaton/issues/616")
 public class CdiBeanCaseTaskResolutionTest extends AbstractFoxPlatformIntegrationTest {
 
   @Deployment(name="pa1")

